probe-rs-cli-util/src/argument_handling.rs

```diff
@@ -65,11 +62,11 @@ fn remove_arguments_test() {
     ];
 
     let mut arguments = vec![
+        "--chip-description-path=kek".to_string(),
+        "--chip-description-path".to_string(),
         "--chip=kek".to_string(),
```

Comment:
```suggestion
        "--chip-description-path".to_string(),
        "kek".to_string(),
```

Reply:
Oh, I see my mistake, but the suggestion is not correct either :) Will fix later :)